test/perf/benchmarks/assets/compiler.test.ps1

```diff
@@ -1149,7 +1149,7 @@ function Start-PSBootstrap {
             $Deps = @()
             if ($environment.IsLinux -and $environment.IsUbuntu) {
                 # Build tools
-                $Deps += "curl", "g++", "cmake", "make"
+                $Deps += "curl", "g++", "make"
```

Comment:
do we need g++ and make?

Reply:
This file is a test file, so the content in it doesn't matter.